cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/home/tingzhi/Downloads/ns-3-dev-git/.waf-1.7.16-9ca17eb492c97b689870b4ff9db75880/waflib/Tools/c_config.py", line 460, in run_c_code\n    bld.compile()\n  File "/home/tingzhi/Downloads/ns-3-dev-git/.waf-1.7.16-9ca17eb492c97b689870b4ff9db75880/waflib/Build.py", line 188, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 46340304: cxx test.cpp -> test.cpp.1.o}\n[\'/usr/bin/g++\', \'-O3\', \'-g\', \'-Wall\', \'-Werror\', \'-march=native\', \'-fstrict-overflow\', \'-DHAVE___UINT128_T=1\', \'-DINT64X64_USE_128=1\', \'-DHAVE_SYS_TYPES_H=1\', \'-DHAVE_SYS_STAT_H=1\', \'-DHAVE_DIRENT_H=1\', \'-DHAVE_STDLIB_H=1\', \'-DHAVE_GETENV=1\', \'-DHAVE_SIGNAL_H=1\', \'-DHAVE_PTHREAD_H=1\', \'-DHAVE_STDINT_H=1\', \'-DHAVE_INTTYPES_H=1\', \'../test.cpp\', \'-c\', \'-o\', \'test.cpp.1.o\']\n'
